.changeset/stupid-drinks-boil.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
We switched to changeset-bot instead of failing CI when change file is not detected. You don't need to create empty change file for changes that do not required to be mentioned in changelog (or for package that is not released)

Reply:
Removed changeset